CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.23.1] - 2022-09-15
+
+Fields in `EventMetadata` now also store name of a type specified in the pallet's source code.
+
```

Comment:
```suggestion
```

Let's remove these; Love the general idea, but there are a bunch of commits that aren't listed, and we don't want to commit to a release date :)

Reply:
Could add an `[unnreleased]` section at the top as per convention

Reply:
I pondered that briefly, but it only makes sense if the other commits are also listed, else it could potentially lead to things being missed!

Might be worth starting to do that at some point though for sure!